aiohttp/test_utils.py

```diff
@@ -326,16 +326,16 @@ class TestClient:
 
     :type app: aiohttp.web.Application
 
-    :param protocol: the aiohttp.web application passed to create_test_server
+    :param protocol: http or https
 
-    :type app: aiohttp.web.Application
+    :type protocol: str
 
     TestClient can also be used as a contextmanager, returning
     the instance of itself instantiated.
     """
 
     def __init__(self, app, protocol="http"):
-        self._app = app
+        self.app = app
```

Comment:
Yes, was about to make a comment about this.

Very often in tents it's helpful to get access to the `app` eg. to get the db connection and check data has changed as expected.